### PR TITLE
feat(examples): add html, data, and custom fonts

### PR DESCRIPTION
**Pull request by elnaz**
@pon @leore 

https://github.com/lob/lob-api/issues/1505

### CODE REVIEW

**Comment by pon** [comment]
LGTM

File: examples/check.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 example_address = lob.Address.create(
     name = 'Joe Smith',
+    description = 'Joe - Home',
+    metadata = {
+        'group': 'Members'
```
Comment (leore) [inline examples/check.py]: does group have to be in quotes to work?

Comment (pon) [inline examples/check.py]: Yeah for python dictionaries you do

**Comment by leore** [comment]
LGTM

**Pull request merged**